Mathlib/Topology/Bornology/Absorbs.lean

```diff
@@ -0,0 +1,247 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Data.Set.Pointwise.SMul
+import Mathlib.Topology.Bornology.Constructions
+
+#align_import analysis.locally_convex.basic from "leanprover-community/mathlib"@"f2ce6086713c78a7f880485f7917ea547a215982"
+
+/-!
+# Definitions of `Absorbs` and `Absorbent`
```

Comment:
Can we have a mathematically readable name?

Reply:
Could you please update it as you prefer?

Reply:
Something like
```suggestion
# Absorption of sets
```